components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/AS2ClientManager.java

```diff
@@ -35,6 +35,7 @@
 import org.apache.camel.component.as2.api.util.EntityUtils;
 import org.apache.camel.component.as2.api.util.SigningUtils;
 import org.apache.camel.util.ObjectHelper;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Avoid using 3rd party libraries for string stuff. Camel already have or the JDK have as well